docker/Dockerfile.binary

```diff
@@ -0,0 +1,26 @@
+FROM docker.io/clux/muslrust:stable as cargo-build
+WORKDIR /usr/src/gpdata
+
+# Copy and Build Code
+COPY . .
+RUN env CARGO_PROFILE_RELEASE_DEBUG=1 cargo build --target x86_64-unknown-linux-musl --release
+
+RUN \
+    cd .. && \
+    git clone https://github.com/gnosis/regex-stream-split.git && \
+    cd regex-stream-split && \
+    git checkout edc88224612b9e151c334fa6d3a7d20575d83836 && \
+    cargo build --target x86_64-unknown-linux-musl --release
+
+# Extract Binary
+FROM docker.io/alpine:latest
+
+# Handle signal handlers properly
+RUN apk add --no-cache tini
+COPY --from=cargo-build /usr/src/regex-stream-split/target/x86_64-unknown-linux-musl/release/regex-stream-split /usr/local/bin/regex-stream-split
+COPY --from=cargo-build /usr/src/gpdata/target/x86_64-unknown-linux-musl/release/gpdata /usr/local/bin/gpdata
+COPY --from=cargo-build /usr/src/gpdata/user_data.json /usr/local/data/gpdata/user_data.json
```

Comment:
Super nitpicky but the [Linux FHS](https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.pdf) suggests putting application specific data under `share`, so `/usr/local/share/gpdata/user_data.json`.

It really doesn't make a difference though , so no need to change it.